.travis.yml

```diff
@@ -3,6 +3,7 @@ language: go
 sudo: required
 
 go:
+  - "1.14"

```

Comment:
Remove this change. 1.14 is too too old.

Reply:
Do you want to consider writing 1.16 here?